From dbb6a9dfc9eecb1f07461bb0730b59bec6412008 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Tue, 5 Oct 2021 19:18:48 -0700
Subject: [PATCH 01/17] SystemUI: Reduce screenshot dismiss delay to 3 seconds

3 seconds is enough in most cases. Keeping the overlay around for too
long forces the user to dismiss it manually, which is a regression
compared to the Android 10's screenshot notification UI.

Change-Id: I62a87ba880d3d11717e465f87267037b06b1e02d
---
 .../android/systemui/screenshot/LegacyScreenshotController.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/screenshot/LegacyScreenshotController.java b/packages/SystemUI/src/com/android/systemui/screenshot/LegacyScreenshotController.java
index 9f9120b98..84eb86792 100644
--- a/packages/SystemUI/src/com/android/systemui/screenshot/LegacyScreenshotController.java
+++ b/packages/SystemUI/src/com/android/systemui/screenshot/LegacyScreenshotController.java
@@ -98,7 +98,7 @@ public class LegacyScreenshotController implements InteractiveScreenshotHandler
     // From WizardManagerHelper.java
     private static final String SETTINGS_SECURE_USER_SETUP_COMPLETE = "user_setup_complete";
 
-    static final int SCREENSHOT_CORNER_DEFAULT_TIMEOUT_MILLIS = 6000;
+    static final int SCREENSHOT_CORNER_DEFAULT_TIMEOUT_MILLIS = 3000;
 
     private final WindowContext mContext;
     private final FeatureFlags mFlags;
-- 
2.51.1

